Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -30,9 +30,18 @@ class AppStoreConnectService {
     }
 
     func inviteBetaTesterToGroups(with options: InviteBetaTesterOptions) throws -> BetaTester {
-        let output = try InviteTesterOperation(options: options).execute(with: requestor).await()
+        let sdkBetaTester = try InviteTesterOperation(options: options).execute(with: requestor).await()
 
-        return BetaTester(with: output)
+        let getBetaTesterOperation = GetBetaTesterOperation(
+            options: .init(
+                id: sdkBetaTester.id,
+                email: nil
+            )
+        )
+
+        let output = try getBetaTesterOperation.execute(with: requestor).await()
+
+        return BetaTester(output: output)
```

Comment:
We can reduce this a lot I think:

```suggestion
        let getBetaTesterOptions = GetBetaTesterOperation.Options(id: sdkBetaTester.id, email: nil)

        return try GetBetaTesterOperation(options: getBetaTesterOptions).execute(with: requestor).await()
```

Reply:
Sorry you'll need to map to an actual beta tester, but you get the idea 😄